app/controllers/channels/profiles_controller.rb

```diff
@@ -13,14 +13,28 @@ def edit
 
   def update
     authorize resource
-    update!
+
+    if resource.update(permitted_params)
+      flash[:notice] = t('channels.profiles.update.success')
```

Comment:
O que tu acha de fazer assim?
```Ruby
flash[:notice] = t('success', scope: 'channels.profiles.update')
```

 Talvez ficasse mais fácil de ler, já que a mensagem que a gente quer está logo no início, e só no `scope`, a gente tem o caminho para a tradução.